Calendar/CalLogHelpers/TimelineFunctions.ps1

```diff
@@ -61,8 +61,7 @@ function FindFirstMeeting {
 }
 
 function BuildTimeline {
-    $script:TimeLineOutput = @()
-    #$script:TimeLineOutput
+    $script:TimeLineOutput = New-Object System.Collections.Generic.List[System.String]
```

Comment:
If you are going to use a `System.Collections.Generic.List[System.String]` you need to switch the add method to be `$script:TimeLineOutput.Add($value)` otherwise, PowerShell will switch it back to a `System.Array`

![image](https://github.com/microsoft/CSS-Exchange/assets/22776718/f7ba85a0-d053-4713-b0ff-118bc4f986bf)

Reply:
Having issues with this...   <Don't Check in>

Reply:
Then we should change the constructor of the object back to `@()`.